fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -333,16 +339,8 @@ public void init(Analyzer analyzer) throws UserException {
         computePartitionInfo();
         computeTupleState(analyzer);
 
-        /**
-         * Compute InAccurate cardinality before mv selector and tablet pruning.
```

Comment:
Keep this logic and supply the 'StatsRecursiveDerive'

Reply:
done